server/src/controllers/Venue/resolver.ts

```diff
@@ -102,6 +104,48 @@ export class VenueResolver {
     @Arg('_onlyUsedForAuth', () => Int) _onlyUsedForAuth: number,
   ): Promise<{ id: number }> {
     // TODO: handle deletion of non-existent venue
+    const users = await prisma.users.findMany({
+      where: {
+        user_events: {
+          every: {
+            event: {
+              canceled: false,
+              ends_at: { gt: new Date() },
+              venue_type: events_venue_type_enum.Physical,
+            },
+          },
+        },
+      },
+    });
+
+    const venue = await prisma.venues.findUniqueOrThrow({
+      where: { id: id },
+    });
+    const chapterId = venue.chapter_id;
+
+    const unsubscribeOptions = (userId: number) =>
+      chapterUnsubscribeOptions({
```

Comment:
Wouldn't the `eventUnsubscribeOptions` be better here?

Reply:
Let's go, I am not the only one who feels that way 🤣. https://github.com/freeCodeCamp/chapter/issues/2262

Reply:
Well yes and no. I initially made suggestion to search for the affected users via the `event_users`, but then I realized single user can be in multiple events having this venue what would send them multiple emails stating _Events hosted at ${venue.name}_.

Now when I think about it, maybe it would be better to send one email per event. Or still send one email per user, but specifically include in the email all events affected, to which user is RSVPed.

Reply:
> Or still send one email per user, but specifically include in the email all events affected, to which user is RSVPed.

This, please. We should take any opportunity to keep the number of emails low.

Reply:
oh, I thought this commit sorted this https://github.com/freeCodeCamp/chapter/pull/2233/commits/f0dbf3b454613c2ee7963c2c2596dd986ce52032

Did I misunderstand something?